test/parallel/test-whatwg-encoding-textdecoder.js

```diff
@@ -2,6 +2,10 @@
 'use strict';
 
 const common = require('../common');
+
+if (!common.hasIntl)
+  common.skip('missing Intl');
```

Comment:
With the exception of UTF-16BE, this test should work even when ICU is not enabled.